src/vcpkg/commands.add-version.cpp

```diff
@@ -116,6 +119,8 @@ namespace
                                  "",
                                  "can't obtain SHA for port {package_name}");
     DECLARE_AND_REGISTER_MESSAGE(AddVersionPortDoesNotExist, (msg::package_name), "", "{package_name} does not exist");
+    DECLARE_AND_REGISTER_MESSAGE(EmptyCommitMessage, (), "", "Error: The specified commit message must be not empty.");
+    DECLARE_AND_REGISTER_MESSAGE(GitCommitFailed, (), "", "Error: Failed to commit the changes. The git output is: ");
```

Comment:
```suggestion
    DECLARE_AND_REGISTER_MESSAGE(EmptyCommitMessage, (), "", "commit message must not be empty.");
    DECLARE_AND_REGISTER_MESSAGE(GitCommitFailed, (), "", "failed to commit changes. The git output is: ");
```
We shouldn't prefix these messages with `Error:` instead use `msg::print_error` that will append it automatically.